docs/core/project-sdk/msbuild-props.md

```diff
@@ -0,0 +1,201 @@
+
+The `ExcludeAssets` attribute specifies which assets belonging to the package specified by `PackageReference` should not be consumed.
+
+The `PrivateAssets` attribute specifies which assets belonging to the package specified by `PackageReference` should be consumed but not flow to the next project. The `Analyzers`, `Build`, and `ContentFiles` assets are private, by default, when this attribute is not present.
+
+The :::no-loc text="assets"::: attributes can contain one or more of the following values. Separate multiple values with a semicolon `;` character.
+
+| Value | Effect |
+| - | - |
+| `Compile` | The contents of the *lib* folder are available to compile against. |
+| `Runtime` | The contents of the *runtime* folder are distributed. |
+| `ContentFiles` | The contents of the *contentfiles* folder are used. |
+| `Build` | The props/targets in the *build* folder are used. |
+| `Native` | The contents from native assets are copied to the *output* folder for runtime. |
+| `Analyzers` | The analyzers are used. |
+| `None` | None of the assets are used. (This value cannot be combined.) |
+| `All` | All assets are used. (This value cannot be combined.) |
+
+### PackageTargetFallback
+
+The `PackageTargetFallback` property lets you specify a set of compatible targets to be used when restoring packages. It's designed to allow packages that use the dotnet [TxM (Target x Moniker)](/nuget/schema/target-frameworks) to operate with packages that don't declare a dotnet TxM. If you don't add the `PackageTargetFallback` property, then all the packages your project depends on must also have a dotnet TxM.
+
+The following example specifies fallbacks for all targets:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wpa81+wp8</PackageTargetFallback>
+  </PropertyGroup>
+</Project>
+```
+
+The following example uses the `Condition` attribute to specify fallbacks only for the `netcoreapp2.1` target:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageTargetFallback Condition="'$(TargetFramework)'=='netcoreapp2.1'">$(PackageTargetFallback);portable-net45+win8+wpa81+wp8</PackageTargetFallback>
+  </PropertyGroup>
+</Project>
+```
+
+### RuntimeIdentifier
+
+The `RuntimeIdentifier` property lets you specify a single [runtime identifier (RID)](../rid-catalog.md) for the project. The RID enables publishing a self-contained deployment.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <RuntimeIdentifier>ubuntu.16.04-x64</RuntimeIdentifier>
+  </PropertyGroup>
+</Project>
+```
+
+Use `RuntimeIdentifiers` (plural) instead if you need to publish for multiple runtimes. However, `RuntimeIdentifier` can provide faster builds when only a single runtime is required.
+
+### RuntimeIdentifiers
```

Comment:
This probably needs additional clarity. At first read it sounds like it might work the same way as `TargetFramework`, causing an inner-loop to run to build multiple times/publish multiple outputs. That's not the case and `RuntimeIdentifiers` is really only used at restore time to ensure the right assets are in the graph.